README.md

```diff
@@ -1,10 +1,34 @@
 # go-tuf
 
-[![build](https://github.com/theupdateframework/go-tuf/workflows/build/badge.svg)](https://github.com/theupdateframework/go-tuf/actions?query=workflow%3Abuild) [![Coverage Status](https://coveralls.io/repos/github/theupdateframework/go-tuf/badge.svg)](https://coveralls.io/github/theupdateframework/go-tuf) [![PkgGoDev](https://pkg.go.dev/badge/github.com/theupdateframework/go-tuf)](https://pkg.go.dev/github.com/theupdateframework/go-tuf) [![Go Report Card](https://goreportcard.com/badge/github.com/theupdateframework/go-tuf)](https://goreportcard.com/report/github.com/theupdateframework/go-tuf)
+![ci](https://github.com/theupdateframework/go-tuf/actions/workflows/ci.yml/badge.svg) [![Coverage Status](https://coveralls.io/repos/github/theupdateframework/go-tuf/badge.svg)](https://coveralls.io/github/theupdateframework/go-tuf) [![PkgGoDev](https://pkg.go.dev/badge/github.com/theupdateframework/go-tuf)](https://pkg.go.dev/github.com/theupdateframework/go-tuf) [![Go Report Card](https://goreportcard.com/badge/github.com/theupdateframework/go-tuf)](https://goreportcard.com/report/github.com/theupdateframework/go-tuf)
 
 This is a Go implementation of [The Update Framework (TUF)](http://theupdateframework.com/),
 a framework for securing software update systems.
 
+## News
```

Comment:
Maybe instead of "News", we should call this "Backwards-compatibility"?

Reply:
hey, I thought of it more as giving users a heads up of the situation so they are not surprised by the upcoming changes. 

Would it be enough if I add an explicit note about the backwards-compatibility in the details section - i.e. the two versions will not be backward compatible so if someone wants to use the existing version, they should pin their version to v0.7.0?

Reply:
Personally I wouldn't call a section "news" unless we plan to continuously update it.

I think it would be better to have an explicit, loud, statement that the API is in flux and folks using <= 0.7.0 must beware

```
# API instability

**Note**: go-tuf's API is in flux and backwards compatibility with version 0.7.0 and earlier will _**not**_ be maintained.

We recommend production users of go-tuf pin their dependency so that changes in main do not cause disruptions.

For more details on the changes, and the rationale for making them, see issue #foo
```

Reply:
Thanks for the suggestion! Updated the message 💯